src/test/java/com/redhat/devtools/intellij/common/utils/DateHelperTest.java

```diff
@@ -81,4 +81,11 @@ public void shouldOnlyPrintDaysIfDifferenceIsXXXDays() {
         String res = DateHelper.humanizeDate(startTime, endTime);
         assertEquals(res, "374 d ");
     }
+
+    @Test
+    public void shouldOnlyPrint0SecondsIfStartIsOlderThanEnd() {
```

Comment:
test is badly name and I think we are missing the test for the negative test

Reply:
Suggestion for the name? What do you mean by negative test? That is the negative test, if you pass a start time that is older than the end time you have a negative result

Reply:
so this test should be renamed as `shouldOnlyPrint0SecondsIfStartIsEqualToEnd` and add a new test `shouldOnlyPrint0SecondsIfStartIsOlderThanEnd` when start = end + 1(s|m|h)